app/src/main/res/layout/solution_summary.xml

```diff
@@ -11,119 +11,156 @@
-          android:layout_marginEnd="8dp"
-          android:background="@drawable/reveal_solution_brown_background"
-          android:padding="4dp"
-          android:text="@string/reveal_solution"
-          android:textColor="@color/white"
-          app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          android:visibility="@{viewModel.isSolutionRevealed? View.GONE : View.VISIBLE}"
-          app:layout_constraintTop_toBottomOf="@+id/solution_title" />
-
-        <FrameLayout
-          android:id="@+id/expand_list_icon"
+        <TextView
+          android:id="@+id/solution_correct_answer"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
+          android:fontFamily="sans-serif"
           android:layout_marginStart="8dp"
-          android:layout_marginTop="16dp"
-          android:layout_marginEnd="8dp"
-          android:layout_marginBottom="8dp"
-          android:minWidth="48dp"
-          android:minHeight="48dp"
-          app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintHorizontal_bias="0.966"
-          app:layout_constraintStart_toEndOf="@+id/solution_title"
-          app:layout_constraintTop_toTopOf="parent"
-          android:visibility="@{viewModel.isSolutionRevealed? View.VISIBLE : View.GONE}"
-          app:layout_constraintVertical_bias="0.0">
-
-          <ImageView
-            android:id="@+id/chapter_list_drop_down_icon"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center_vertical"
-            android:contentDescription="@{@string/show_hide_solution_list(viewModel.solutionSummary)}"
-            android:src="@{isListExpanded? @drawable/ic_arrow_drop_up_black_24dp : @drawable/ic_arrow_drop_down_black_24dp}" />
-        </FrameLayout>
-      </androidx.constraintlayout.widget.ConstraintLayout>
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="16sp" />
+      </LinearLayout>
 
       <LinearLayout
-        android:id="@+id/solution_container"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:padding="8dp"
-        android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}">
+        android:orientation="vertical">
 
         <TextView
-          android:id="@+id/solution_correct_answer"
+          android:id="@+id/solution_summary_label"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:fontFamily="sans-serif-medium"
+          android:layout_gravity="start"
+          android:fontFamily="sans-serif"
+          android:layout_marginTop="12dp"
+          android:textStyle="bold"
```

Comment:
May i know why this is required?

Reply:
I tried to make it san-serif bold by using android font-family option. I removed it and used san-serif-black.